src/components/app-content-renderer/virtual-assistant.js

```diff
@@ -0,0 +1,27 @@
+import { useFlag } from '@unleash/proxy-client-react';
+import React from 'react';
+import { ScalprumComponent } from '@scalprum/react-core';
+
+import './virtual-assistant.scss';
+import useChrome from "@redhat-cloud-services/frontend-components/useChrome";
+
+function VirtualAssistant() {
+    const showVirtualAssistant = useFlag('platform.landing-page.virtual-assistant');
+    const chrome = useChrome();
+
+    if (!chrome.isBeta || !showVirtualAssistant) {
```

Comment:
I would change this condition slightly. This would surface the VA on all beta envs including production. Can we have something like if is production stable do not show it all, otherwise use the FF.

Reply:
I could rewrite this to make it to be more strict for now, but the plan is to release on prod/beta by early December.
The idea is to have the flag off until we're ready for that.

Reply:
Disabled it on prod. I'm keeping the beta, because I only want to show it on preview for now.

Reply:
Thanks for catching that up! 
I recall some of these where attributes before :thinking:

Reply:
Done, thanks